lib/Rex/Fork/Task.pm

```diff
@@ -41,7 +41,7 @@ sub start {
 
     # only allow this if no parallelism is given.
     # with parallelism active it doesn't make sense.
-    if ( $Rex::WITH_EXIT_STATUS && Rex::Config->get_parallelism == 1 ) {
+    if ( $Rex::WITH_EXIT_STATUS && Rex::Config->get_parallelism eq '1' ) {
```

Comment:
@krimdomu: for me it was needed because the `-t` CLI argument was converted to string from the previous integer. As I was not 100% sure about that part, I'm totally up for better suggestions to handle it. If you could try to revert this single line and test locally, and can come up with a more suitable solution, I would greatly appreciate it!